saleor/webhook/payloads.py

```diff
@@ -69,6 +76,8 @@
     "phone",
 )
 
+CHANNEL_FIELDS_IN_CHECKOUT_ORDER_PAYLOADS = ("slug", "currency_code")
```

Comment:
Why not simply `CHANNEL_FIELDS`? Don't see any different env with this variable. If we will introduce a separate method for building payload for events like `channel_created`, then we could rename it to something like `CHANNEL_FIELDS_FOR_EXTERNAL_OBJECTS`